### PR TITLE
demo for comment ans

### PR DESCRIPTION
**Pull request by bhaktisavani-artoon**
we have added one variable

### CODE REVIEW

File: question.js
Comment (jeminbhalani) [inline question.js:1]: #comment2: why you added this code i don't think it's really needed

File: question.js [hunk 1]
```diff
@@ -0,0 +1 @@
+let Question = "we are learning how to answere on the client's comment"
```
Comment (jeminbhalani) [inline question.js]: #Specific line comment3: Why you added this line

**Review by jeminbhalani — COMMENTED** [review]
Please check this

File: question.js (hunk 1 above)
Comment (bhaktisavani-artoon) [reply question.js]: for Question

File: question.js
Comment (bhaktisavani-artoon) [reply question.js:1]: I have added this code to implement new feature

**Pull request merged**